lib/helpers/getSpecificationObject.js

```diff
@@ -0,0 +1,71 @@
+const parser = require('swagger-parser');
+const createSpecification = require('./createSpecification');
+const specHelper = require('./specification');
+const parseApiFile = require('./parseApiFile');
+const filterJsDocComments = require('./filterJsDocComments');
+const convertGlobPaths = require('./convertGlobPaths');
+
+function isEmptyObject(obj) {
```

Comment:
What do you think about this : `const isEmptyObject = obj => Object.keys(obj).length` ?

Reply:
Not quite, the idea is to find properties with empty objects, not whether properties exist.
if your suggestion is `isEmptyObject2`:

```
const obj = {foo: {}}
undefined
isEmptyObject(obj)
true
isEmptyObject2(obj)
1
```

1 is truthy, which is false for what's the check intention.